libraries/@mattermost/rnutils/android/src/oldarch/RNUtilsModule.kt

```diff
@@ -66,12 +66,12 @@ class RNUtilsModule(context: ReactApplicationContext) :
 
     @ReactMethod(isBlockingSynchronousMethod = true)
     fun deleteDatabaseDirectory(databaseName: String?, shouldRemoveDirectory: Boolean): WritableMap {
-        return implementation.deleteDatabaseDirectory(databaseName, shouldRemoveDirectory)
+        return implementation.deleteDatabaseDirectory()
     }
 
     @ReactMethod(isBlockingSynchronousMethod = true)
     fun renameDatabase(databaseName: String?, newDatabaseName: String?): WritableMap {
-        return implementation.renameDatabase(databaseName, newDatabaseName)
+        return implementation.renameDatabase()
```

Comment:
This change in particular seems really off. How does the inner function understand it has to use the database name and the new database name? Is this some kotlin magic or a bug?

Reply:
this is the implementation doing nothing, is just here in order to fulfill the requirements of the library having the same methods on both platforms.